health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -210,6 +215,28 @@ const CreateChecklist = () => {
     const local = [];
     let activeCounters = { top: 0 }; // Track active question counts at each level
 
+    // Precompute common values once
+    let checklistTypeTemp = checklistType.toUpperCase().replace(/ /g, "_");
+    if (checklistTypeCode) checklistTypeTemp = checklistTypeCode;
+    let roleTemp = role.toUpperCase().replace(/ /g, "_");
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'roleTemp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)